pyproject.toml

```diff
@@ -35,9 +35,9 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-    "cffi ~= 1.15.1",
+    "cffi >= 1.15.1",
     "pytest>=3.8",
-    "filelock ~= 3.12.2",
+    "filelock >= 3.12.2",
     "setuptools ~= 67.8.0; python_full_version >= '3.12.0b1'", # FIXME: remove when cffi supports directly python 3.12
```

Comment:
Can this requirement on setuptools be loosened or removed? The strict pin is causing issues with packaging over at conda-forge - https://github.com/conda-forge/staged-recipes/pull/24569 (partly because conda-forge doesn't have setuptools=67.8.0 for some reason [here](https://anaconda.org/conda-forge/setuptools/files)):

```
WARNING: failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform linux-64: {MatchSpec("setuptools[version='>=67.8.0,<67.9.dev0']")}
Encountered problems while solving:
  - nothing provides requested setuptools >=67.8.0,<67.9.dev0

Could not solve for environment specs
The following package could not be installed
└─ setuptools >=67.8.0,<67.9.dev0  does not exist (perhaps a typo or a missing channel).
```

Looking on PyPI, [`cffi=1.16.0`](https://pypi.org/project/cffi/1.16.0/) should have Python 3.12 wheels already.

Reply:
It looks good, happy to merge this requirements change on `filelock`